src/plugins/orangehrmPerformancePlugin/Dao/PerformanceReviewDao.php

```diff
@@ -0,0 +1,147 @@
+     */
+    private function getReviewListQueryBuilderWrapper(PerformanceReviewSearchFilterParams $performanceReviewSearchFilterParams): QueryBuilderWrapper
+    {
+        $qb = $this->createQueryBuilder(PerformanceReview::class, 'performanceReview');
+        $qb->leftJoin('performanceReview.employee', 'employee');
+        $qb->leftJoin('performanceReview.reviewers', 'reviewer');
+        $qb->leftJoin('reviewer.employee', 'reviewerEmployee');
+        $qb->leftJoin('reviewer.group', 'reviewGroup');
+        $qb->leftJoin('performanceReview.jobTitle', 'jobTitle');
+        $qb->leftJoin('performanceReview.department', 'subUnit');
+
+        $qb->andWhere($qb->expr()->eq('reviewGroup.name', ':reviewGroupName'))
+            ->setParameter('reviewGroupName', 'Supervisor');
+
+        if (!is_null($performanceReviewSearchFilterParams->getSupervisorId())) {
+            $qb->andWhere($qb->expr()->eq('reviewerEmployee.empNumber', ':supervisorEmpNumber'))
+                ->setParameter('supervisorEmpNumber', $performanceReviewSearchFilterParams->getSupervisorId());
+        }
+
+        if (!is_null($performanceReviewSearchFilterParams->getEmpNumber())) {
+            $qb->andWhere($qb->expr()->eq('performanceReview.employee', ':empNumber'))
+                ->setParameter('empNumber', $performanceReviewSearchFilterParams->getEmpNumber());
+        }
+
+        if (!is_null($performanceReviewSearchFilterParams->getStatusId())) {
+            $qb->andWhere($qb->expr()->eq('performanceReview.statusId', ':statusId'))
+                ->setParameter('statusId', $performanceReviewSearchFilterParams->getStatusId());
+        } elseif ($performanceReviewSearchFilterParams->isExcludeInactiveReviews()) {
+            $qb->andWhere($qb->expr()->neq('performanceReview.statusId', ':statusId'))
+                ->setParameter('statusId', 1);
+        }
+
+        if (!is_null($performanceReviewSearchFilterParams->getFromDate())) {
+            $qb->andWhere($qb->expr()->gte('performanceReview.dueDate', ':fromDate'))
+                ->setParameter('fromDate', $performanceReviewSearchFilterParams->getFromDate());
+        }
+
+        if (!is_null($performanceReviewSearchFilterParams->getToDate())) {
+            $qb->andWhere($qb->expr()->lte('performanceReview.dueDate', ':toDate'))
+                ->setParameter('toDate', $performanceReviewSearchFilterParams->getToDate());
+        }
+
+        if (is_null($performanceReviewSearchFilterParams->getFromDate()) &&
+            is_null($performanceReviewSearchFilterParams->getToDate())
+        ) {
+            $currentYear = $this->getDateTimeHelper()->getNow()->format('Y');
+            $qb->andWhere($qb->expr()->gte('performanceReview.dueDate', ':fromDate'))
+                ->setParameter('fromDate', "$currentYear-01-01");
+            $qb->andWhere($qb->expr()->lte('performanceReview.dueDate', ':toDate'))
+                ->setParameter('toDate', "$currentYear-12-31");
+        }
```

Comment:
you can check both null at the API class and set default value there as well

Reply:
changed to check at API